restli-server/src/test/java/com/linkedin/restli/server/resources/fixtures/DefaultConstructorArgResource.java

```diff
@@ -0,0 +1,57 @@
+/*
+   Copyright (c) 2012 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package com.linkedin.restli.server.resources.fixtures;
+
+import com.linkedin.restli.server.TestRecord;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.resources.CollectionResourceTemplate;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@RestLiCollection(name="defaultConstructorArgResource",
```

Comment:
Just for clarity, leave a javadoc saying that the purpose of this class is to test injecting resources via package-private constructors (but worded more better)

Reply:
Added.